src/components/Banners/SiteBanner.tsx

```diff
@@ -4,9 +4,9 @@ import Icon from "@leafygreen-ui/icon";
 import IconButton from "@leafygreen-ui/icon-button";
 import parse from "html-react-parser";
 import Cookies from "js-cookie";
-import { getJiraTicketUrl } from "constants/externalResources";
 import { GetSpruceConfigQuery } from "gql/generated/types";
 import { GET_SPRUCE_CONFIG } from "gql/queries";
+import { jiraLinkify } from "utils/string/jiraLinkify";
```

Comment:
nit. for consistency can we use 
```
import { string } from "utils";

const { jiraLinkify} = "string";
```

Reply:
jiraLinkify actually can't be exported from index without running into a dep cycle. jiraLinkify imports externalResources and the contents of string and externalResources are exported from utils

Reply:
Woah, This is the first time i've seen a dependency cycle in javascript.